native_client/BUILD

```diff
@@ -67,14 +60,11 @@ tf_cc_shared_object(
             "kiss_fft130/_kiss_fft_guts.h",
             "kiss_fft130/tools/kiss_fftr.h",
             "ds_version.h"] +
-            select({
-                "//tensorflow:windows": DECODER_SOURCESWIN,
-                "//conditions:default": DECODER_SOURCES,
-            }),
+            DECODER_SOURCES,
     # -Wno-sign-compare to silent a lot of warnings from tensorflow itself,
     # which makes it harder to see our own warnings
     copts = select({ 
-        "//tensorflow:windows": [],
+        "//tensorflow:windows": ["/w"],
```

Comment:
Maybe worth moving the `-Wno-sign-compare` comment next to the use, and also commenting that `-fvisibility=hidden` is default on this platform, to make it easier in the future